extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -62,20 +63,22 @@ amp-story[standalone][desktop] {
 
 .i-amphtml-story-background {
   position: absolute !important;
-  top: 0 !important;
-  right: 0 !important;
-  bottom: 0 !important;
-  left: 0 !important;
+  top: -150px !important;
```

Comment:
This seems arbitrary; is there any reasoning behind the number?  If so, can we add a comment?  If not.... can we add a comment why it doesn't matter?

Reply:
This was merged from #12629 here, but I forgot to push again for the diff to update. Sorry about that!